### ISSUE TITLE
input options - only get child items of parent

### ISSUE THREAD

**Issue by FickleLife** [benjamin-vauchel/SuperBoxSelect#5]
I can't seem to find what input options there are for superBoxSelect in the documentation. I there a way where by you can only select resources under a certain parent? 

Thanks for a great addon too.

**Comment by shaunmorrison** [comment]
Yes, I can't see this in the documentation either. Not familiar with ModX TV Input Options. Assume you have to enter '@select id =`3`' or something like that.

**Comment by shaunmorrison** [comment]
Figured it out: '@SELECT id FROM modx_site_content WHERE parent=3' in the input options and then set output options to be 'delimiter', then add a comma – now it's ready to use with getResources.

**Comment by shaunmorrison** [comment]
Found an issue that it only pulls in immediate children of the specified ID. Sub-descendants aren't being used. May be something wrong with my '@select' though.
